x/vesting/keeper/hooks.go

```diff
@@ -0,0 +1,168 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package keeper
+
+import (
+	"errors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	govv1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+	vestingtypes "github.com/evmos/evmos/v14/x/vesting/types"
+)
+
+var _ govtypes.GovHooks = Hooks{}
+
+// Hooks wrapper struct for the vesting keeper
+type Hooks struct {
+	k Keeper
+}
+
+// Hooks returns the wrapper struct for vesting hooks
+func (k Keeper) Hooks() Hooks {
+	return Hooks{k}
+}
```

Comment:
These lines are opaque to the module. Probably we can move them to an utils package at the root of the project.

Reply:
Those are the hooks definitions and they're inside of the `hooks.go` file and only used/referenced from there? IMO they should not be moved..